client/src/components/_common/AppIcon/AppIcon.jsx

```diff
@@ -3,11 +3,31 @@ import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import Icon from '_common/Icon';
 import './AppIcon.scss';
+import iconStyles from '../../../styles/trumps/icon.css';
+import iconFontsStyles from '../../../styles/trumps/icon.fonts.css';
 
-const AppIcon = ({ appId }) => {
+const doesClassExist = (className, stylesheets) => {
+  for (let stylesheet of stylesheets) {
+    //Required to make this work with Jest/identity-obj-proxy
+    if (typeof stylesheet === 'object') {
+      if (stylesheet[className]) {
+        return true;
+      }
+    } else if (typeof stylesheet === 'string') {
+      if (stylesheet.includes(`.${className}::before`)) {
+        return true;
+      }
+    }
+  }
+  return false;
+};
```

Comment:
Posing the question: do we think this utility function belongs here in this file, as that's its only current use, or under `client/src/utils`?

Reply:
I can move it into utils if you think it'll be useful in the future for other files.